sqlite/sqlite-bundled/build.gradle

```diff
@@ -76,6 +82,14 @@ configurations {
             attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage, Usage.NATIVE_LINK))
         }
     }
+    // Configuration for producing a shareable directory that includes SQLite amalgamation sources.
+    sqliteSources {
```

Comment:
side question: what do we want to do with sources of sqlite that be bundle? should they ship in source jars?

Reply:
@danysantiago , wdyt? 
imo, might be nice for completeness but the only API seen by consumer is the sqlite driver impl created by us (they don't get direct access to sqlite though it might be possible to load it maybe).